scriptworker/cot/verify.py

```diff
@@ -1102,6 +1102,7 @@ def _get_additional_github_releases_jsone_context(parent_link, decision_link):
     jsone_context = {
         'now': parent_link.task['created'],
         'as_slugid': lambda x: task_ids.get(x, task_ids['default']),
+        'taskId': task_ids['decision'],
```

Comment:
This change also adds a 2nd conflicting `taskId` (see line 1111)

Reply:
With the conflicting `taskId` removed, this looks correct.